pkg/kuttlctl/cmd/errors.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kuttl/pkg/test"
+)
+
+var (
+	errorsExample = `  # Asserts "errors" against a $KUBECONFIG cluster the values defined in the assert file.
+  kubectl kuttl errors <path/to/errorsfile.yaml>`
+)
+
+// newErrorsCmd returns a new initialized instance of the errors sub command
+func newErrorsCmd() *cobra.Command {
+	timeout := 5
+	namespace := "default"
+
+	errorsCmd := &cobra.Command{
+		Use:     "errors",
+		Short:   "Asserts the declared errors state to NOT be true.",
+		Long:    `Asserts the declared errors state provided as an argument to not be true in the $KUBECONFIG cluster. Valid arguments are a YAML file, URL to a YAML file.`,
+		Example: errorsExample,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("one file argument is required")
```

Comment:
Check is for one or more, but error message is about exactly one?

Reply:
the check is for exactly 0... and the error message seems consist with assert and accurate.  at least 1 file is required.

Reply:
Let me try rephrasing: the message saying `one is required` is in my mind equivalent to `exactly one is required`. While in reality as you say in another comment, more than one is also fine.

Reply:
makes sense... I'll update!